tests/h/views/main_test.py

```diff
@@ -69,8 +129,16 @@ def test_it_extracts_username_from_account_id(self, pyramid_request):
 
     @pytest.fixture
     def routes(self, pyramid_config):
+        pyramid_config.add_route('activity.search', '/search')
         pyramid_config.add_route('activity.user_search', '/user/{username}')
         pyramid_config.add_route('stream', '/stream')
+        pyramid_config.add_route('stream_atom', '/stream.atom')
+        pyramid_config.add_route('stream_rss', '/stream.rss')
+
+
+@pytest.fixture
+def render_app(patch):
```

Comment:
You can put this fixture inside the `TestStreamUserRedirect` class since it isn't used by any test outside of that class.